README.md

```diff
@@ -285,6 +296,9 @@ packages:
   - package: calogica/dbt_expectations
     version: [">=0.5.0", "<0.6.0"]
 
+  - package: fivetran/amazon_ads
+    version: [">=0.1.0", "<0.2.0"]
```

Comment:
We also need to include the source package as a dependency

Reply:
Done.